wrappers/react-native/example/__tests__/detox-environment.js

```diff
@@ -1,20 +1,21 @@
-const { DetoxCircusEnvironment, SpecReporter, WorkerAssignReporter } = require('detox/runners/jest-circus');
+/*
+ * Copyright 2022 - MATTR Limited
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const { DetoxCircusEnvironment, SpecReporter, WorkerAssignReporter } = require('detox/runners/jest');
 const { FixturesEnvName, readFixtureFiles } = require('../__fixtures__/fixture-loader');
 
 class CustomDetoxEnvironment extends DetoxCircusEnvironment {
   constructor(config, context) {
     super(config, context);
-
-    // Can be safely removed, if you are content with the default value (=300000ms)
-    // this.initTimeout = 300000;
-    this.initTimeout = 900000;
-
-    // This takes care of generating status logs on a per-spec basis. By default, Jest only reports at file-level.
-    // This is strictly optional.
-    this.registerListeners({
-      SpecReporter,
-      WorkerAssignReporter,
-    });
```

Comment:
Do longer needed in mentioned in the detox migration guide.
https://wix.github.io/Detox/docs/guide/migration/#custom-jest-environment